gwsumm/triggers.py

```diff
@@ -101,7 +102,7 @@
     ETG_READ_KW[name] = {
         'format': 'ligolw',
         'tablename': name,
-        'use_numpy_dtype': True,
+        'use_numpy_dtypes': True,
```

Comment:
@duncanmmacleod Just checking that `use_numpy_dtype` ==> `use_numpy_dtypes` is not a typo

Reply:
No, that is the focus of d7536a6305fcdfc20b126ce249b753c02890496b

Reply:
Ok, so by "no" you mean "correct" 😉

Reply:
Correct (sorry).